File: Bubble Sort + Selection Sort + Insertion Sort/js/BubbleSort.js

let arr = [7, 8, 3, 1, 2]; 

for(let i=0; i<arr.length-1; i++) { 

    for(let j=0; j<arr.length-1-i; j++) { 
        

        // swap 
        if(arr[j] > arr[j+1]) { 
            let temp = arr[j];
            arr[j] = arr[j+1];
            arr[j+1] = temp;
        }
    }
}

console.log("Sorted Array : ", arr)

File: Bubble Sort + Selection Sort + Insertion Sort/py/BubbleSort.py

arr = [7, 8, 3, 1, 2]

n = len(arr)

# outer loop for passes 
for i in range(n-1):
    
    # inner loop for comparing different elements
    for j in range(n - 1 - i):
        if(arr[j] > arr[j+1]):
            
            # swap if elements are in wrong order
            arr[j], arr[j+1] = arr[j+1], arr[j]
            
print("Sorted Array : ", arr)